spec/rspec/mocks/partial_double_spec.rb

```diff
@@ -622,6 +622,24 @@ class << self
           end
         end
 
+        context "on a class with a twice-aliased `new`" do
+          it 'uses the method signature from `#initialize` for arg verification' do
+            if RUBY_VERSION.to_i < 3
+              pending "Failing due to Ruby 2's Method#== being too strict"
+            end
```

Comment:
```suggestion
          it 'uses the method signature from `#initialize` for arg verification', :if => RUBY_VERSION.to_i > = 3 do
```
Typically we use meta data rather than pending in spec, and flipping the conditional here should also fix the 1.8.7 error (which apparently supports this :joy:)

Reply:
Right, thanks, I applied it and fixed the `> =`.